### PR TITLE
Added areas, regions, and superregions

### PR DESCRIPTION
**Pull request by That-Guy-Github1**
divided gensokyo and the moon into 6 super regions, 26 regions and 469 states

### CODE REVIEW

**Review by Hecatia-Lapislazuli — CHANGES_REQUESTED** [review]
Seems mostly good, only thing i'd ask to be changed is area names with apostrophes in them - it might work? but is best to avoid non-standard characters (i.e. only use A-Z, a-z, -, _, which is the Regex [0-9a-zA-Z\\\-\\\_])

Also, localization is missing.

**Review by ComradeFlan — COMMENTED** [review]
All the new areas, regions and superregions are missing localizations. This gets my approval once they are added here too

**Comment by That-Guy-Github1** [comment]
@Hecatia-Lapislazuli localization added and apostrophes removed

File: localisation/touhou_map_l_english.yml [hunk 1]
```diff
@@ -3,18 +3,33 @@
...
+ south_foot_of_the_mountain_region: "South Foot of the Mountains"
  youkai_mountain_region: "Youkai Mountain"
- former_hell_region: "Former Hell"
- hell_region: "Hell"
+ west_former_hell_region: "Western Former Hell"
+ east_former_hell_region: "Eastern Former Hell"
+ central_former_hell_region: "Central Former Hell"
+ oni_former_hell_region: "South Former Hell"
+ lower_hell: "Lower Hell"
+  upper_hell: "Lower Hell"
```
Comment (Hecatia-Lapislazuli) [inline localisation/touhou_map_l_english.yml]: There is an extra space on this line

File: localisation/touhou_map_l_english.yml [hunk 2]
```diff
@@ -197,9 +212,770 @@
...
+Spirit_Riverbank: "Spirit Riverbank"
+Spirit_Riverbank_adj: "Spirit Riverbank"
+Misty_Inn: "Misty Inn"
+Misty_Inn_adj: "Misty Inn"
+Hells_Docks: "Hell's Docks"
+Hells_Docks_adj: "Hell's Docks"
+Underworld_Gate: "Underworld Gate"
+Underworld_Gate_adj: "Underworld Gate"
+Youkai_Hotspring: "Youkai Hotspring"
+Youkai_Hotspring_adj: "Youkai Hotspring"
```
Comment (Hecatia-Lapislazuli) [inline localisation/touhou_map_l_english.yml]: These are all missing a space on the first line

File: localisation/touhou_map_l_english.yml [hunk 3]
```diff
@@ -197,9 +212,770 @@
...
  ###SUBCONTINENT###
- gensokyo_superregion: "Gensokyo"
+ youkai_mountain: "youkai mountain Area"
+  Hell: "Hell"
+  former_hell: "former_hell"
```
Comment (Hecatia-Lapislazuli) [inline localisation/touhou_map_l_english.yml]: One too many spaces

Comment (Hecatia-Lapislazuli) [inline localisation/touhou_map_l_english.yml]: should end in `_superregion` and be uncapitalized

File: localisation/touhou_map_l_english.yml [hunk 4]
```diff
@@ -3,18 +3,33 @@
...
- gensokyo_region: "Heart of Gensokyo"
+ Eastern_forest: "East Forest"
+ west_forest_reigion: "West Forest"
+ bamboo_forest_region: "Bamboo Forest"
+ Great_Hakurei_Barrier_reigion: "Great Hakurei Barrier"
  netherworld_region: "Netherworld"
  gensokyan_forest_region: "The Great Forest"
  orbital_region: "Orbit of Gensokyo"
  makai_region: "Makai"
+ kappa_delta: "West Foot of the Mountains"
```
Comment (Hecatia-Lapislazuli) [inline localisation/touhou_map_l_english.yml]: Should end with `_region`

File: localisation/touhou_map_l_english.yml [hunk 5]
```diff
@@ -3,18 +3,33 @@
...
+ east_former_hell_region: "Eastern Former Hell"
+ central_former_hell_region: "Central Former Hell"
+ oni_former_hell_region: "South Former Hell"
+ lower_hell: "Lower Hell"
+  upper_hell: "Lower Hell"
+ sanzu_river: "sanzu river"
+ Animal_realm_reigion : "Animal Realm"
+ Great_border: "Great Borderlands"
  moon_region: "Moon"
+ misty_lake: "Misty Lake"
```
Comment (Hecatia-Lapislazuli) [inline localisation/touhou_map_l_english.yml]: Should end with `_region`
Also shouldn't have capitalization for the entry (i.e. should be `great_border_region`, similarly with `Animal_realm_reigion` should be `animal_realm_reigion`)

**Review by Hecatia-Lapislazuli — CHANGES_REQUESTED** [review]
Hmm, missed it in the first review, but removing regions will break events and decisions... I suppose is somewhat fine, given that a lotta events and decisions (and missions) are broken anyways.

Also some unstandardized region naming stuff, have added mentioned them now.

I'll take a further look later today with how to handle it. May just comment out instances of it currently.

Would also like to see further localization, but not expecting that of you. Can be handled later on.

Regardless of that though...
All that's needed now is just the issues i left comments on with capitalization and missing postfixes... oh and also indentation. After that's handled should be ready to be merged in.

**Comment by That-Guy-Github1** [comment]
formatting and capitalization issues fixed for localization, area, regions and superregions

**Review by Hecatia-Lapislazuli — APPROVED** [review]

**Comment by Hecatia-Lapislazuli** [comment]
How renaming areas was handled will break missions even more by changing already existing areas, but eh, this change is better done now rather than later.

**Pull request merged**